src/Microsoft.Health.SqlServer/Features/Health/SqlServerHealthCheck.cs

```diff
@@ -20,24 +20,31 @@ public class SqlServerHealthCheck : IHealthCheck
     {
         private readonly ILogger<SqlServerHealthCheck> _logger;
         private readonly SqlConnectionWrapperFactory _sqlConnectionWrapperFactory;
+        private readonly RetrySqlAction _retrySqlAction;
 
-        public SqlServerHealthCheck(SqlConnectionWrapperFactory sqlConnectionWrapperFactory, ILogger<SqlServerHealthCheck> logger)
+        public SqlServerHealthCheck(
+                                    SqlConnectionWrapperFactory sqlConnectionWrapperFactory,
```

Comment:
nit: this is personal preference, but I typically just indent only 1 level in this case. I don't know if we have a style enforced anywhere haha